standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/dbinstall/AbstractDatabase.java

```diff
@@ -0,0 +1,302 @@
+package org.apache.hadoop.hive.metastore.dbinstall;
```

Comment:
Isn't `hive-testutils` module a better place for keeping these classes since they are not only used in the `metastore-server`?

Reply:
Yes I felt that the metastore isn’t the right place for this class but didn't know where should it be, I’ll move it into the hive-testutils module.

Reply:
The `hive-testutils` module seems to be the right place for this classes we should be careful though not to create circular dependencies between hive-testutils and metastore modules.